source/api/sdk.bs

```diff
@@ -1955,8 +1955,8 @@ namespace api
         end function
 
         ' Gets items based on a query.
-        function GetItemsByQuery(id as string, params = {} as object)
-            req = APIRequest(Substitute("/users/{0}/items", id), params)
+        function GetItemsByQuery(params = {} as object)
```

Comment:
The use of /items API call is already elsewhere in the SDK, ~ line 422. This one was left in place until the other locations were updated, which you've done here 🤘 

That means we no longer need this GetItemsByQuery() and the spots that use it should be updated to instead call

api.items.Get(params)